shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -113,7 +113,20 @@
                                       blink::CustomAccessibilityActionUpdates actions) {
   if (accessibility_bridge_) {
     accessibility_bridge_->UpdateSemantics(std::move(update), std::move(actions));
+    if (semantics_available_callback_) {
+      semantics_available_callback_();
+      semantics_available_callback_ = nullptr;
+    }
+  }
+}
+
+// |shell::PlatformView|
+bool PlatformViewIOS::RegisterSemanticsAvailableCallback(fml::closure closure) {
+  semantics_available_callback_ = std::move(closure);
+  if (!semantics_available_callback_ || !accessibility_bridge_) {
+    return false;
   }
+  return true;
```

Comment:
This could be:

`return semantics_available_callback_ && accessibility_bridge_;`

However, unless I'm misunderstanding, this doesn't seem to follow the documented contract (which is to return whether or not the callback was successfully registered)?

Reply:
I think my documentation and implementation got out of sync.  I'm going to update the docs, because I don't think I have a good way to force the tree to rebuild.

Reply:
And if I try to track whether it's been built there are thread safety issues at play.